nav2_smac_planner/src/node_hybrid.cpp

```diff
@@ -115,9 +117,7 @@ void HybridMotionTable::initDubin(
   projections.emplace_back(delta_x, -delta_y, -increments);  // Right
 
   // Create the correct OMPL state space
```

Comment:
State space reinitialization could be only avoided at this point, if the only changed thing was the num_angle_quantization. This is a very small / unlikely benefit for the extra code it would take handling it in my opinion.